website/docs/r/service_acl_entries_v1.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Defines a set of Fastly ACL entries that can be used to populate a service ACL.  This resource will populate an ACL with the entries and will track their state.
 
+Note: You must plan / apply the ACL definition along with the variable/locals first. Once provisioned you can then add in the fastly_service_acl_entries_v1 resource to populate the ACL container with entries. 
```

Comment:
```suggestion
You must plan/apply the [ACL definition](https://registry.terraform.io/providers/fastly/fastly/latest/docs/resources/service_v1#nestedblock--acl) (along any variables/locals first) in your service configuration. Once the ACL has been provisioned you can then add in the fastly_service_acl_entries_v1 resource to populate the ACL container with entries. 
```

Reply:
Also can we validate if we even need this addition or if we could update the examples to use [`depends_on`](https://www.terraform.io/docs/language/meta-arguments/depends_on.html) in the `fastly_service_acl_entries_v1` block example below?

e.g.

```terraform
resource "fastly_service_acl_entries_v1" "entries" {
  service_id = fastly_service_v1.myservice.id
  acl_id = {for d in fastly_service_v1.myservice.acl : d.name => d.acl_id}[var.myacl_name]
  entry {
    ip = "127.0.0.1"
    subnet = "24"
    negated = false
    comment = "ALC Entry 1"
  }
  depends_on = [fastly_service_v1.myservice]
}
```

Reply:
I've not used depends_on before. Looks like it came into play in TF 0.13 which was GA Aug 2020. There's been a few versions since and now TF 1.0 so it might still be worth referencing for customers using older versions.

Reply:
Hey, @alpagot

> Looks like it came into play in TF 0.13 which was GA Aug 2020

What you're referring to is:
https://www.terraform.io/docs/language/meta-arguments/depends_on.html

> Version note: Module support for `depends_on` was added in Terraform 0.13, and previous versions can only use it with resources.

In this case, we're talking about `resource`, not `module`, and `depends_on` was added quite some time ago. So I'm +1 with @Integralist 's suggestion.